pkg/project/api/v1beta1/types.go

```diff
@@ -31,7 +31,7 @@ type ProjectStatus struct {
 type Project struct {
 	kapi.TypeMeta   `json:",inline"`
 	kapi.ObjectMeta `json:"metadata,omitempty"`
-	DisplayName     string `json:"displayName,omitempty"`
+	//DisplayName, Description are Annotations (in kapi.ObjectMeta struct)
```

Comment:
It's a breaking API change when you remove something from an API (since clients look for it, wont find it, and will break).  We can change the API by having a new version (in this case v1beta2), since no clients use it yet, and in that version, we can remove DisplayName